eng/tools/lint-diff/src/lint-diff.ts

```diff
@@ -102,11 +103,25 @@ async function runLintDiff(
   baseBranch: string,
   compareSha: string,
 ) {
-  const [beforeList, afterList, affectedSwaggers] = await getRunList(
-    beforePath,
-    afterPath,
-    changedFilesPath,
-  );
+  let beforeList, afterList, affectedSwaggers;
+  try {
+    [beforeList, afterList, affectedSwaggers] = await getRunList(
+      beforePath,
+      afterPath,
+      changedFilesPath,
+    );
+  } catch (error) {
+    process.exitCode = 1;
+    console.log(error);
+
+    if (error instanceof SpecModelError) { 
```

Comment:
I wonder if we should have a JS helper function `isSpecModelError()`, like I created here?

https://github.com/Azure/azure-rest-api-specs/blob/c9d1fa5a56b1306828b347f655133d6c398180cd/.github/shared/src/exec.js#L21-L35

I can't remember exactly why I needed this.  Maybe because I'm typedef-ing an existing error, while SpecModelError doesn't need this, because it's a class we're defining?

Reply:
`SpecModelError` is a class so you can use `instanceof`

ExecError is typedef'd https://github.com/Azure/azure-rest-api-specs/blob/main/.github/shared/src/exec.js#L22

Reply:
I'll try to replace ExecError with a type in a separate PR.  When I started writing the shared JS code, I think I was too anti-class.